packages/builder/builder-shared/src/utils.ts

```diff
@@ -116,3 +117,21 @@ export const isCssModules = (filename: string, modules: CssModules) => {
   }
   return true;
 };
+
+export const getCssModulesAutoRule = (
+  config: CssModules,
+  disableCssModuleExtension = false,
+) => {
+  if (
+    config === false ||
```

Comment:
`output.cssModules: false` and `output.cssModules.auto: false` seem to be the same, we can only support `output.cssModules.auto: false` o make it simpler.